cmd/thanos/query.go

```diff
@@ -122,6 +127,8 @@ func registerQuery(app *extkingpin.App) {
 	fileSDInterval := extkingpin.ModelDuration(cmd.Flag("store.sd-interval", "Refresh interval to re-read file SD files. It is used as a resync fallback.").
 		Default("5m"))
 
+	endpointConfig := extflag.RegisterPathOrContent(cmd, "endpoint.config", "YAML file that contains store API servers configuration. Either use this option or separate endpoint options (endpoint, endpoint.sd-files, endpoint.srict).", extflag.WithEnvSubstitution())
```

Comment:
```suggestion
	endpointConfig := extflag.RegisterPathOrContent(cmd, "endpoint.config", "YAML file that contains set of endpoints (e.g Store API) with optional TLS options. Either use this option or separate endpoint options (endpoint, endpoint.sd-files, endpoint.strict).", extflag.WithEnvSubstitution())
```

Reply:
> Either use this option or separate endpoint options (endpoint, endpoint.sd-files, endpoint.strict)

This line is wrong as we accept separate endpoint options (endpoint, endpoint.sd-files, endpoint.strict) along with new `--endpoint.config`.

My suggestion: 
`YAML file that contains set of endpoints (e.g Store API) with optional TLS options. To enable TLS either use this option or deprecated ones --grpc-client-tls*` WDYT?

Reply:
Perfect 👍🏽